scripts/build/companion_libs/220-ncurses.sh

```diff
@@ -49,7 +49,7 @@ do_ncurses_for_host() {
     # (i.e. in simple cross or native): ncurses may not be needed for
     # host, but we still need them on build to produce 'tic'.
     case "${CT_TOOLCHAIN_TYPE}" in
-        native|cross)   return 0;;
+        native)   return 0;;
```

Comment:
With "native|cross", gdb fails to start with TUI:

$ ./arm-cortex_m4-eabi-gdb -tui
Cannot enable the TUI: error opening terminal [TERM=xterm]

With "native" only it works fine.

So there is some sort of difference here which breaks gdb. It seems ncurses related. Needs further investigation.